tests/fixtures/civic_aid6.yaml

```diff
@@ -0,0 +1,539 @@
+id: civic.aid:6
+type: Statement
+description: L858R is among the most common sensitizing EGFR mutations in NSCLC,
+  and is assessed via DNA mutational analysis, including Sanger sequencing and next
+  generation sequencing methods. Tyrosine kinase inhibitor afatinib is FDA approved
+  as a first line systemic therapy in NSCLC with sensitizing EGFR mutation
+  (civic.EID:2997).
+extensions:
+- name: evidence_ids
+  value:
+    - civic.eid:2997
+    - civic.eid:2629
+    - civic.eid:982
+    - civic.eid:968
+    - civic.eid:883
+    - civic.eid:879
```

Comment:
@ahwagner I wasn't sure how to handle instances where we weren't able to transform a CIViC evidence item associated to the assertion. So I just created an extension to store all evidence items associated. `hasEvidenceItems` is where we would store the ones that could be transformed.